CHANGELOG.md

```diff
@@ -12,6 +12,22 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 22.2.13 (2024-04-30)
```

Comment:
Do we want some additional note/warning somewhere in the changelog that this release is only relevant for damask/archive nodes.

Reply:
I will mark it as a non-latest release, hopefully that is enough.